app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class InvitesController < ApplicationController
+  before_action :set_group
+
+  def create
+    authorize! :manage, @group
+
+    recipient = User.find_by(email: invite_params[:email])
+    invite = @group.invites.build(
```

Comment:
we don't use role in the invite?

Reply:
The role will be used when invite accepting is implemented